docs/install/operator/configuring-eventing-cr.md

```diff
@@ -8,12 +8,38 @@ aliases:
 
 The Knative Eventing operator can be configured with these options:
 
+- [Version Configuration](#version-configuration)
 - [Eventing Configuration by ConfigMap](#eventing-configuration-by-configMap)
 - [Private repository and private secret](#private-repository-and-private-secrets)
 - [Configuring default broker class](#configuring-default-broker-class)
 
 __NOTE:__ Kubernetes spec level policies cannot be configured using the Knative operators.
 
+## Version Configuration
+
+As of 0.16.0, Knative users are able to install Knative Eventing of a specific version via the field `spec.version`. For
```

Comment:
Can we say "administrators" or "operators" -- I think we want to make clear that this is targeted at the "operator" persona rather than the "developer" persona:

![image](https://user-images.githubusercontent.com/7959095/88423182-3b34d100-cda0-11ea-806c-6da326ad5dc2.png)

Reply:
done.